examples/parser/examples.ipynb

```diff
@@ -0,0 +1,763 @@
+{
```

Comment:
In current revision `.ipynb` examples should be stored as python sources formatted using [`py:percent`](https://jupytext.readthedocs.io/en/latest/formats.html#the-percent-format).